_posts/06-01-01-Dependency-Injection.md

```diff
@@ -6,13 +6,11 @@ anchor: dependency_injection
 
 # Injektovanje zavisnosti (Dependency Injection) {#dependency_injection_title}
 
-Sa [Wikipedia](http://en.wikipedia.org/wiki/Dependency_injection)-je:
+Sa [Wikipedia](http://en.wikipedia.org/wiki/Dependency_injection)-e:
 
 > Injektovanje zavisnosti je softverski dizajn patern koji omogućava uklanjanje hardkodovanih zavisnosti
```

Comment:
Da, definitivno je bolje Dependency Injection, to je u principu ista ona priča kao sa framework, design pattern, document root i slično. Ljudi su navikli na te strane termine.